src/mlpack/tests/cf_test.cpp

```diff
@@ -32,15 +32,15 @@ BOOST_AUTO_TEST_CASE(CFGetRecommendationsAllUsersTest)
 {
   // Dummy number of recommendations.
   size_t numRecs = 3;
-  // GroupLens100k.csv dataset has 943 users.
-  size_t numUsers = 943;
+  // GroupLensSmall.csv dataset has 943 users.
```

Comment:
Oops, this was just a typo---I made the dataset smaller, but I forgot to update the comment.